fire/SFMainMod.F90

```diff
@@ -962,7 +999,8 @@ subroutine  cambial_damage_kill ( currentSite )
 
        if (currentPatch%fire == 1) then
           currentCohort => currentPatch%tallest;
-          do while(associated(currentCohort))  
+          do while(associated(currentCohort))
+             currentCohort%cambial_mort = 0.0_r8
```

Comment:
nice, good place to zero

Reply:
on board with the clean up of zeroing...